iml-agent/src/cli.rs

```diff
@@ -496,13 +496,32 @@ fn add_counter_entry(x: impl Counter, t: &mut Table, h: &mut v_hist::Histogram)
     h.add_entry(name, x.count().try_into().unwrap());
 }
 
+fn exe_name() -> Option<String> {
+    Some(
+        std::env::current_exe()
+            .ok()?
+            .file_stem()?
+            .to_str()?
+            .to_string(),
+    )
+}
+
+// FIXME: dedup with iml-manager-cli
```

Comment:
Does this need to be addressed now?

Reply:
I think we can consider this debt for the time being.

Reply:
There is no crate which would be the best place for it. I was thinking about new iml-cli.